core/arch/arm/plat-d02/conf.mk

```diff
@@ -4,8 +4,7 @@ CFG_NUM_THREADS ?= 16
 CFG_CRYPTO_WITH_CE ?= y
 CFG_WITH_STACK_CANARIES ?= y
 CFG_WITH_SOFTWARE_PRNG ?= n
-# Override default size of emulated TrustZone protected SRAM, 384 kB
-CFG_CORE_TZSRAM_EMUL_SIZE ?= 393216
+CFG_CORE_TZSRAM_EMUL_SIZE ?= 524288
```

Comment:
Unfortunately, no. `time xtest 4002` takes 0.2 seconds when size is 512K, vs. 8.1 seconds when size is 448K.